policy-service/src/contract-tests/resources/application-pre-population.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Not interpreted by any java code, simply used by reference in the population.resources section
+population:
+  userProvider: std
+  users:
+  - userId: noResourceRulesUser
+    auths:
+      auth
+    roles:
+      role
+
+  - userId: test-user-id
+    auths:
+      auth
+    roles:
+      role
```

Comment:
How do these two users differ?

Reply:
users are the same, but the polices and resources that apply to them differ, you could technically have one user but it makes it a bit clearer to have 2